Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -295,11 +295,9 @@ public enum RenderBlockContent: Equatable {
     
     /// A type the describes an aside style.
     public struct AsideStyle: Codable, Equatable {
-        private static let specialDisplayNames: [String: String] = [
-            "nonmutatingvariant": "Non-Mutating Variant",
-            "mutatingvariant": "Mutating Variant",
-            "todo": "To Do",
-        ]
+        private static let knownDisplayNames: [String: String] = Dictionary(
+            uniqueKeysWithValues: Markdown.Aside.Kind.allCases.map { ($0.rawValue.lowercased(), $0.displayName) }
```

Comment:
This moves ownership of the `displayName` over to Swift-Markdown where the existing ownership of asides is. Moving forward, we'll be able to add new supported asides with a single PR.

A bit of a discussion about this here: https://github.com/apple/swift-markdown/pull/96#discussion_r1065263050